File: Data Structures & Algorithms/length-of-last-word/submission-4.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int r = s.length()-1;
        while (s[r] == ' ') {
            r--; 
        }
        
        int l = r; 
        while (l >= 0 && s[l] != ' ') {
            l--;
        }
        cout << l << ' ' << r;
        return (r-l); 

    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-4.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        hs = set(nums)

        starts = []
        for num in hs:
            if num-1 not in hs:
                starts.append(num)
        
        res = 0
        for num in starts:
            n = num
            curr = 1
            while n+1 in hs:
                curr+=1
                n=n+1
            res = max(res, curr)
        return res; 

        